src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -7583,28 +7583,6 @@
         }
       }
     },
-    "csp": {
```

Comment:
Apologies for the noise... resolving conflicts ended up writing a different order of the schemas :/